script/testing/junit/src/FunctionsTest.java

```diff
@@ -159,4 +159,16 @@ public void testLower() throws SQLException {
         checkStringFunc("lower", "str_a_val", true, null);
     }
 
+    @Test
+    public void testVersion() throws SQLException {
+        String sql = String.format("SELECT version() AS result");
+
+        Statement stmt = conn.createStatement();
+        ResultSet rs = stmt.executeQuery(sql);
+        boolean exists = rs.next();
+        assert(exists);
+        checkStringRow(rs, new String[]{"result"}, new String[]{"NoisePage 1.0.0"});
```

Comment:
Make it check that it matches with what is in version.h. Otherwise this test will break every time we update the version.

Reply:
I added a C++ test file (system_functions_test.cpp) that checks what's in version.h, but I couldn't find a way to get the value in the header file to the java file. Should I just take out this test then?

Reply:
You can take out the test. We are going to replace the Junit tests anyway.